### PR TITLE
doc(INVT-CPC-1007): update read me to include ssh keys

### PR DESCRIPTION
**Pull request by nic5694**
updated README with information about SSH Keys

JIRA: link to jira ticket
## Context:
Added SSH Key Infroamtion
## Changes
- Added steps to keep passphrase to avoid entering passphrase at each command that requires permission

### CODE REVIEW

**Review by Nmazza1 — DISMISSED** [review]
Documentation is Documentation. 👍

File: README.md [hunk 1]
```diff
@@ -379,4 +379,15 @@ git stash pop
 ```
 
 ---
+## SSH Keys
+Note: This section is if you are using **SSH keys to clone your repo** and have already generated your GitHub SSH keys and ran your keygen in your terminal (console of your computer)
 
+ 
+A common problem you face when setting the SSH key on mac is when you restart your computer the ssh key makes you authenticate with your passphrase.
```
Comment (RichardDanon) [inline README.md:386]: Small writing TIp.

..................... on the Mac is that when you restart your computer .....

**Review by RichardDanon — APPROVED** [review]
View and Green

File: README.md (hunk 1 above)
Comment (nic5694) [reply README.md:386]: Small Tip: when nitpicking please add "Nitpick:"

**Review by RichardDanon — APPROVED** [review]

**Review by Nmazza1 — APPROVED** [review]
10/10 Linguistics

**Pull request merged**